manifest.php

```diff
@@ -38,7 +38,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '29.7.2',
+    'version'     => '29.7.3',
```

Comment:
You need to update to a feature version since you require a version with a breaking change : 
```
'version'     => '29.8.0',

'requires'    => array(
        //...
        'tao'        => '>=23.0.0',
     
```